runatlantis.io/docs/repo-level-atlantis-yaml.md

```diff
@@ -148,6 +148,11 @@ See [Custom Workflow Use Cases: Adding extra arguments to Terraform commands](cu
 See [Custom Workflow Use Cases: Custom init/plan/apply Commands](custom-workflows.html#custom-init-plan-apply-commands)
 
 ### Terragrunt
+
+Terragrunt declares its module dependencies explicitly, so it is possible to generate an `atlantis.yaml` file by examining all the dependencies in your project.
+
+To take advantage of this, you can use the tool [terragrunt-atlantis-config](https://github.com/transcend-io/terragrunt-atlantis-config).
```

Comment:
Can you reword this to say that *if* you don't want to create the `atlantis.yaml` file yourself, you can use this tool to automatically generate it. Also add a note that this tool is a community project and not maintained by the Atlantis team.

Reply:
I reworded it. Please let me know what you think of the updated text!